tests/Unit/Css/CssDocumentTest.php

```diff
@@ -437,15 +441,52 @@ public function provideCssWithoutNonConditionalAtRules(): array
      * @dataProvider provideValidAtCharsetRules
      * @dataProvider provideInvalidAtCharsetRules
      */
-    public function discardsValidOrInvalidAtCharsetRule(string $css): void
+    public function discardsValidOrInvalidAtCharsetRuleNotInDebugMode(string $css): void
     {
-        $subject = new CssDocument($css);
+        $subject = new CssDocument($css, false);
 
         $result = $subject->renderNonConditionalAtRules();
 
         self::assertSame('', $result);
     }
 
+    /**
+     * @test
+     *
+     * @param string $css
```

Comment:
Generally, please let's not have PHPDoc annotations that add nothing to what the native type declarations already communicate. (We probably still need to removed the redundant annotations IIRC.)

Reply:
Agree.  We've already removed the CI check that tried to enforce unnecessary DocBlock.  This was a result of copypasta.

I have hopefully now removed unnecessary DocBlock annotations from all methods touched by this change (whether existing or new).

There are definitely others that need cleaning up.